geode-core/src/main/java/org/apache/geode/internal/cache/persistence/PersistenceAdvisorImpl.java

```diff
@@ -510,18 +510,19 @@ public PersistentMembershipView getMembershipView() {
   public boolean checkMyStateOnMembers(Set<InternalDistributedMember> replicates)
       throws ReplyException {
     PersistentStateQueryResults remoteStates = getMyStateOnMembers(replicates);
+    Set<InternalDistributedMember> copyOfReplicates = new HashSet<>();
+    copyOfReplicates.addAll(replicates);
```

Comment:
You can shrink this down to one statement:
```
Set<InternalDistributedMember> copyOfReplicates = new HashSet<>(replicates);
```

Reply:
all fixed